src/key.rs

```diff
@@ -809,9 +809,9 @@ impl KeyPair {
         KeyPair::new(SECP256K1, rng)
     }
 
-    /// Serializes the key pair as a secret key byte value.
+    /// Gets the secret bytes for this key pair.
     #[inline]
-    pub fn serialize_secret(&self) -> [u8; constants::SECRET_KEY_SIZE] {
+    pub fn to_bytes(&self) -> [u8; constants::SECRET_KEY_SIZE] {
```

Comment:
Since this is key *pair* I think it'd be clearer to name this method `secret_bytes()` (Was also thinking of `to_` prefix but that usually isn't used for getters.)

Reply:
Weak ACK -- I think it would be clearer to label this method with `secret` somehow.

Reply:
Too trigger happy with the the `to_` conversions huh :) Use `secret_bytes` as suggested. Cheers.